web/src/SidebarKeyboardShortcuts.test.tsx

```diff
@@ -61,3 +67,11 @@ it("navigates backwards no wrap", () => {
   fireEvent.keyDown(document.body, { key: "k" })
   expect(fakeHistory.location.pathname).toEqual("/init")
 })
+
+it("triggers update", () => {
+  let items = twoResourceView().resources.map(res => new SidebarItem(res))
+  let sks = shortcuts(items, "")
+  expect(triggered).toEqual(false)
+  fireEvent.keyDown(document.body, { key: "Enter" })
+  expect(triggered).toEqual(true)
```

Comment:
does this test assert WHICH element got triggered? that seems important

Reply:
also, worth a test for the new behavior (that clicking on a selected element triggers update -- and maybe also that clicking on an unselected element selects it but does not trigger update)?